knimin/handlers/barcode_util.py

```diff
@@ -1,12 +1,42 @@
 #!/usr/bin/env python
 from tornado.web import authenticated
+from tornado.escape import json_encode
+from tornado import gen, concurrent
 from knimin.handlers.base import BaseHandler
 from datetime import datetime
+import requests
+import functools
+from qiita_client import QiitaClient
 
 from knimin import db
 from knimin.lib.constants import survey_type
 from knimin.lib.mail import send_email
 from knimin.handlers.access_decorators import set_access
+from knimin.lib.configuration import config
+
+
+def get_qiita_client():
```

Comment:
I couldn't find a test for this.

Reply:
I guess the benefit isn't really clear to me for this method. But can add something to exercise it.

Reply:
IMOO the benefit is that you can check that the method actually connects to Qiita; which is missing from the added tests. Is this intentional?

Reply:
Yes, as it would require either storing credentials in travis which is annoying or greatly complicate the CI environment. This is executed at process start, so if `QiitaClient` doesn't initiate, it will be blazingly obvious too.